.rubocop_todo.yml

```diff
@@ -31,14 +31,20 @@ Metrics/BlockLength:
 Metrics/MethodLength:
   Max: 24
 
-# Offense count: 4
+# Offense count: 5
 # Configuration parameters: Max.
 RSpec/ExampleLength:
   Exclude:
     - 'spec/features/create_apo_spec.rb'
     - 'spec/features/create_collection_spec.rb'
     - 'spec/features/create_object_no_files_spec.rb'
     - 'spec/features/reaccession_via_preassembly_spec.rb'
+    - 'spec/features/sdr_deposit_spec.rb'
+
+# Offense count: 1
+RSpec/LeakyConstantDeclaration:
+  Exclude:
+    - 'spec/features/sdr_deposit_spec.rb'
```

Comment:
looking at https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/LeakyConstantDeclaration this *seems* like something we should care about (but looking at the spec I don't understand exactly what it is caring about)

Reply:
I guess it's about the login helper class. Can get rid of this exclusion if the class's name is destroyed via `const_remove` in an `after` hook, if memory serves.

Reply:
well, that's obtuse.

Reply:
Not a concern as this is only temporary until we can proxy credentials

Reply:
Not a concern as this is only temporary until we can proxy credentials